cmd/root.go

```diff
@@ -164,6 +164,43 @@ func BuildRoot(f factory.Factory, excludePlugins bool) *cobra.Command {
 	persistentFlags := rootCmd.PersistentFlags()
 	globalFlags = flags.SetGlobalFlags(persistentFlags)
 
+	rootCmd.SetUsageTemplate(`Usage:{{if .Runnable}}
+  {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
+  {{.CommandPath}} [command]{{end}}{{if gt (len .Aliases) 0}}
+  
+Aliases:
+  {{.NameAndAliases}}{{end}}{{if .HasExample}}
+  
+Examples:
+{{.Example}}{{end}}{{if .HasAvailableSubCommands}}
+  
+Available Commands:{{range .Commands}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
+  
+Flags:
+{{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
+  
+Global Flags:
+{{.InheritedFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasHelpSubCommands}}
+
+Additional help topics:{{range .Commands}}{{if .IsAdditionalHelpTopicCommand}}
+  {{rpad .CommandPath .CommandPathPadding}} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableSubCommands}}
+
+{{- if (and .HasAvailableSubCommands) -}}
+{{- range .Commands -}}
+{{- if (and .HasSubCommands (eq .Name "run"))}}
+
+Additional run commands:
+{{- range .Commands}}
+  {{rpad (printf "'%s'" .CommandPath) .CommandPathPadding}} {{.Short}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+{{- end}}
+
+Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
```

Comment:
I'm not sure if this might be confusing as people would expect that you can run `devspace run my-command --help`. Maybe we should remove this or move it above the `Additional run commands` section